api/envoy/extensions/filters/common/dependency/v3/dependency.proto

```diff
@@ -0,0 +1,37 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.common.dependency.v3;
```

Comment:
I think this location is fine. I debated this a bit with myself, since in world in which this is only a build time dependency, it's arguably not part of the xDS APIs proper. However, I think when we have dynamic filters, such as LuaJIT and Wasm, it would be reasonable to deliver this information alongside the filter bytecode. In this case this proto will be referenced by the Lua/Wasm extension wrapper protos, and it needs to be in the xDS API somewhere.

Arguably it could also be under `envoy.extensions.filters.http.common`, since this set of definitions is specific to HTTP. If you think this will be mostly identical modulo some enum values when we do network filers, here is fine though.

Reply:
Yeah I think network filters would use the same proto.